cmd/main.go

```diff
@@ -68,6 +68,7 @@ func init() {
 	utilruntime.Must(monitoringv1.AddToScheme(scheme))
 	utilruntime.Must(configv1.AddToScheme(scheme))
 	utilruntime.Must(securityv1.AddToScheme(scheme))
+	utilruntime.Must(hypershiftv1beta1.AddToScheme(scheme))
```

Comment:
@Clee2691 @anpingli have we tested this in a non HCP cluster to ensure the operator does not fail to start because maybe it is missing the CRD for the hypershift apis?

Reply:
Isn't that what the e2e tests basically confirm? CLO is able to be installed and send logs even with the hypershift scheme added. On each reconcile, it will try to fetch the `clusterID` and `version` so if it failed, none of those tests would have passed.

Reply:
Tested on regular cluster, the clusterID present correctly. No regression issue.